src/main/java/com/zilen/weather/entity/Weather.java

```diff
@@ -0,0 +1,83 @@
+package com.zilen.weather.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Weather {
+
+    @Id
+    @Column(name = "id")
```

Comment:
Probably, when field name equal to column name, annotation isn't required and can be omitted.